tox.ini

```diff
@@ -21,6 +21,8 @@ envlist =
     isort
     jsonformat
     pytest
+    lint
```

Comment:
Maybe `mdlint` or `markdownlint` to make it more clear what we're linting.

Reply:
Good point, fixed in [97055fa](https://github.com/eiffel-community/eiffel/pull/304/commits/97055fa448a3828fd2c385ea80a91a119d38119a)